### ISSUE TITLE
Local environment trying to load resources from port 80

### ISSUE THREAD

**Issue by aero31aero** [Requarks/wiki#393]
### Actual behavior

I set domain as the default http://localhost and port to serve on as 4000

Visiting localhost:4000 gives: 
![image](https://user-images.githubusercontent.com/8033238/35740955-6d708fd0-085c-11e8-8b60-66291e05b444.png)

Also, when I clicked on go to wiki button, it redirected me to port 80 where I was greeted by my Apache server.

### Expected behavior

WikiJS should honor the port while fetching static resources and in the configuration UI's redirect button.

### Steps to reproduce the behavior

Already told above. Here's the relevant part of the generated config:

```
title: Bits Wiki
host: 'http://localhost'
port: 4000
paths:
  repo: ./repo
  data: ./data
```

I also tried restarting with `node wiki restart` to see if that solves the problem of some initial configuration or first run issue.

**Comment by aero31aero** [comment]
### Note

I just read the docs and the last lines asks me to specify the port number in the host as well. Shouldn't a relative url be used, however, by simply just refactoring 

```js
    script(type='text/javascript', src=appconfig.host + '/js/vendor.js')
    script(type='text/javascript', src=appconfig.host + '/js/app.js')
```

to 

```js
    script(type='text/javascript', src='/js/vendor.js')
    script(type='text/javascript', src='/js/app.js')
```

? Am I missing some design decision?

BTW, awesome work, guys. :smile:

**Comment by NGPixel** [comment]
The `host` was added to support sub-folder installation, but it was never fully completed. This will be addressed in 2.0.
